src/components/NavigatorSearch/modes/messages.ts

```diff
@@ -101,6 +101,14 @@ const messages = defineMessages({
     id: "x/ZVlU",
     defaultMessage: "Product",
   },
+  root: {
+    defaultMessage: "This is root category",
+    id: "6iaXL/",
+  },
+  variant: {
+    defaultMessage: "Variant",
+    id: "OK5+Fh",
+  },
```

Comment:
nitpick: I think we might have string "Variant" somewhere else for translations, so maybe we can re-use it here?